src/libs/actions/Report.ts

```diff
@@ -2440,7 +2440,7 @@ const updatePrivateNotes = (reportID: string, accountID: number, note: string) =
 };
 
 /** Fetches all the private notes for a given report */
-function getReportPrivateNote(reportID: string) {
+function getReportPrivateNote(reportID?: string) {
```

Comment:
Maybe better `string | undefined`?

Reply:
It's the same thing 😄

Reply:
actually not @pac-guerreiro , if you have `?` then its optional to put this param and you can use this functionas 
`getReportPrivateNote()`
but when you do`string | undefined` you cannot do that and you always need to add this parameter but can but the value can be undefined.

Reply:
@kubabutkiewicz Oh I didn't know that! Sorry @cubuspl42 😅 I'm going to apply your suggestion!

Reply:
Yes, that's the difference. I would suggest to go with `string | undefined`. IMO it's just better to go with `| undefined` unless we're sure we care about the optional argument syntax.